way_to_home/tests/unittests/custom_user/testmodel.py

```diff
@@ -0,0 +1,116 @@
+"""
+CustomUser Model Test
+========================
+This module provides complete testing for all CustomUser's model functions.
+"""
+
+from django.test import TestCase
+from custom_user.models import CustomUser
+
+
+class CustomUserTestCase(TestCase):
+    """TestCase for providing CustomUser's model testing"""
+
+    def setUp(self):
+        """ Method that provides preparation before testing CustomUser model's features. """
+        self.custom_user = CustomUser.objects.create(
+            id=111,
+            password='1111',
+            email='user@mail.com',
+            google_token='example_google_token_1111',
+            phone_number='+3801111',
+            is_active=True
+        )
+        self.custom_user.save()
+
+    def test_str(self):
+        """ Test of the __str__() method """
+        custom_user = CustomUser.objects.get(id=self.custom_user.id)
+        returned_string = custom_user.__str__()
+        expected_string = f'{self.custom_user.id} {self.custom_user.email}'
+        self.assertEqual(returned_string, expected_string)
+
+    def test_to_dict(self):
+        """ Test for checking dictionary that 'to_dict' method return """
+        custom_user = CustomUser.objects.get(id=self.custom_user.id)
+        returned_dict = custom_user.to_dict()
+        expected_dict = {
+            'id': custom_user.id,
+            'email': custom_user.email,
+            'phone_number': custom_user.phone_number
+        }
+        self.assertDictEqual(returned_dict, expected_dict)
+
+    def test_create(self):
+        """ Provide tests for 'create' method of certain CustomUser instance """
+        created_user = CustomUser.create(
+            email='created_user@mail.com',
+            password='2222',
+            google_token='example_google_token_2222',
+            phone_number='+3802222'
+        )
+
+        self.assertIsInstance(created_user, CustomUser)
+        self.assertIsNotNone(CustomUser.objects.get(id=created_user.id))
+
+        created_user = CustomUser.create(
+            email=None,
+            password='',
+            google_token='example_google_token_2222',
+            phone_number='+3802222'
+        )
+        self.assertIsNone(created_user)
+
+    def test_get_by_email(self):
+        """ Provide tests for 'get_by_email' method of certain CustomUser instance """
+        expected_user = CustomUser.objects.get(id=self.custom_user.id)
```

Comment:
```suggestion
        expected_user = CustomUser.objects.get(id=self.custom_user.id)
```
You can also to retrieve with param `email` here.

Reply:
Fixed!